### PR TITLE
added multi-upc/quantity to receieve view. looks ugly, and not merged…

### PR DESCRIPTION
**Pull request by eristow**
… with my other receive changes yet.

### CODE REVIEW

**Comment by angryoctopode** [comment]
Looks good I guess, Im assuming it works.... I did mine a completely different way though.

**Review by angryoctopode — DISMISSED** [review]

**Review by dkwhite200 — DISMISSED** [review]
Looks good. I assume it works.

**Comment by eristow** [comment]
I also decided to finally fix those single quotes warnings when running npm start.

**Review by AriTheGuitarMan — DISMISSED** [review]
The alerts seem to work well. It's pretty simple and if it works, then I wholeheartedly approve! I guess the Logic for the receive inventory was coded out differently, but, again, if it works, it works, so I have no criticism. Good job!

File: modules/inventorymanagements/client/controllers/receive-inventorymanagements.client.controller.js [hunk 1]
```diff
@@ -11,57 +11,84 @@
...
-        alert("You must fill in UPC first");
-      } else if (!$scope.quantity) {
-        alert("You must fill in Quantity first");
-      } else {
-        if($scope.quantity <= 0) {
-          alert("Quantity must be greater than 0");
+      // search for UPC in DB. if there, add quantity. if not, send to create page.
+      // initial check over array of choices for error
+      for (var i = 0; i < $scope.choices.length; i++) {
+        console.log($scope.choices, i, $scope.choices[i]);
```
Comment (dkwhite200) [inline modules/inventorymanagements/client/controllers/receive-inventorymanagements.client.controller.js]: Do we still need this console.log?

**Review by dkwhite200 — COMMENTED** [review]
One comment on code review.

**Review by dkwhite200 — APPROVED** [review]
good

**Pull request merged**